ReverseTicTacToe/UI/ConsoleUI.cs

```diff
@@ -70,7 +194,7 @@ private  void getGameTypeFromUser()
                 userInputStr = ReadLine();
             }
             while(!isValidInput(userInputStr,1,2));
-            m_GameType = (Game.eGameTypes)System.Enum.Parse(typeof(Game.eGameTypes), userInputStr);
+            return (Game.eGameTypes)System.Enum.Parse(typeof(Game.eGameTypes), userInputStr);
         }
 
         public bool isValidInput(string i_UserInputStr, int min, int max)
```

Comment:
Maybe make that method's name more elaborative? by specifying what kind of input

Reply:
I don't understand what you mean here

Reply:
There are multiple input formats in the program, at different stages - board size input, game mode input, makeMove input... so i think the name "isValidInput" is a bit unclear -  what kind of input is this method suppose to validate?